launch/baxter.launch

```diff
@@ -0,0 +1,38 @@
+<launch>
+  <arg name="camera_frame" default="/head_mount_kinect_rgb_optical_frame" />
+
+  <arg name="sim" />
+  <arg if="$(arg sim)" name="point_cloud_topic" default="/mock_point_cloud" />
+  <arg if="$(arg sim)" name="depth_topic" default="/head_mount_kinect/depth/image_raw" />
+  <arg if="$(arg sim)" name="color_topic" default="/head_mount_kinect/rgb/image_raw" />
+
+  <arg unless="$(arg sim)" name="point_cloud_topic" default="/camera/depth_registered/points" />
+  <arg unless="$(arg sim)" name="depth_topic" default="/camera/depth_registered/image" />
+  <arg unless="$(arg sim)" name="color_topic" default="/camera/depth_registered/image_color" />
+
+  <group ns="rapid_pbd"> 
+    <param name="robot" value="baxter" />
+    <node pkg="rapid_pbd" type="baxter_action_node" name="baxter_actuator_server" />
+    <node pkg="rapid_pbd" type="program_execution_node" name="program_executor">
+      <remap from="l_arm_joint_action" to="/robot/limb/left/follow_joint_trajectory" />
+      <remap from="r_arm_joint_action" to="/robot/limb/right/follow_joint_trajectory" />
+      <remap from="move_group" to="/move_group" />
+    </node>
+    <rosparam command="load" file="$(find rapid_pbd)/config/surface_segmentation.yaml" />
+    <node pkg="rapid_pbd" type="surface_segmentation_node" name="surface_segmentation_node" args="$(arg point_cloud_topic)" />
+  </group>
+
+  <node pkg="tf" type="static_transform_publisher" name="world_to_camera" args="0 0 1 0 0 0 base camera_link 10" />
```

Comment:
Move this up to the top of the launch file (after the args but before the first `group ns="rapid_pbd"`) so it's easier to find and customize.